repositories/src/main/scala/cromwell/pipeline/datastorage/dto/TypedValue.scala

```diff
@@ -0,0 +1,54 @@
+package cromwell.pipeline.datastorage.dto
+
+import java.nio.file.{ Path, Paths }
+import play.api.libs.functional.syntax._
+
+import play.api.libs.json.{ Format, JsNull, JsResult, JsValue, Writes }
+
+sealed trait TypedValue
```

Comment:
As i understand we need this converter for mongo. But name TypedValue - is too general.

Reply:
It's not only for mongo, when we will want to change value of node, it would help us to understand the type of node.